website/www/site/content/en/get-started/tour-of-beam.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Tour of Beam"
+title: "An Intro to Beam (Python)"
```

Comment:
I think the plan is to link this to the tour of beam website and add a new page for an intro to Beam (Python).

Reply:
+1 on making a new page, could we just incorporate that into this PR? Should mostly just need to update the references that were added in the original PR https://github.com/apache/beam/commit/96866e6a5cea4464cd595c221e3bb44e5ec48ad4

Reply:
Just to clarify, we have two options here:

1. Make a new page for "An Intro to Beam (Python)".
2. Add this material into our [Python Quickstart](https://beam.apache.org/get-started/quickstart/python/)


I actually prefer 2) since these will all be Python specific. But, because they teach fundamental Beam concepts, it might make sense to introduce a new page.


Thoughts?

Reply:
If we do end up going with (1), I think we need a better name :) 

Maybe `An Interactive overview of Beam`?

Reply:
I'd vote for just renaming this page (so option (1)) - I like `An interactive overview of Beam`

Reply:
(and probably linking in the next steps of the Python Quickstart)